src/scss/buefy.scss

```diff
@@ -1,6 +1,7 @@
 @import "utils/_all";
 
 @import "components/_autocomplete";
+@import "components/_breadcrumb.scss";
```

Comment:
Please remove it and the empty scss file (Bulma already provides it)

Reply:
Forgot to remove it.

Reply:
Done all changes.